tests/test_s3_storage.py

```diff
@@ -79,3 +79,28 @@ class TestStorage(PrivateS3Storage):
     storage = TestStorage()
 
     assert storage.get_path("test.txt") == "http://s3.fastapi.storages/test.txt"
+
+
+@mock_s3
+def test_s3_storage_duplicate_file_names(tmp_path: Path) -> None:
+    s3 = boto3.client("s3")
+    s3.create_bucket(Bucket="bucket")
+
+    tmp_file = tmp_path / "duplicate.txt"
+    tmp_file.write_bytes(b"123")
```

Comment:
You don't need this. You are writing the file multiple times, so this is not doing anything.

Reply:
Change it to `.touch()`